release-notes/topics/operator-220.adoc

```diff
@@ -0,0 +1,14 @@
+[[operator-220-intro]]
+= Automation Platform Operator 2.2
+
+The Ansible Automation Platform Operator provides cloud-native, push-button deployment of new Ansible Automation Platform instances in your OpenShift environment.
```

Comment:
Should we use attributes for the product names here or is that not appropriate for this instance? I haven't done release notes. So maybe it doesn't apply here. 

```suggestion
The {OperatorPlatform} {PlatformVers} provides cloud-native, push-button deployment of new {OperatorPlatform} instances in your OpenShift environment.
```
This would change it to Ansible Automation Platform Operator 2.2, but if you intentionally kept it at "Automation Platform Operator 2.2, please disregard.

Reply:
@jself-sudoku Good call, however, tt doesn't look like the attributes folder is part of the release notes, so changing them here caused problems in the build. Perhaps that should be fixed in the future but in the interest of getting this completed, I'm leaving these as is for this PR. Thanks!